### ISSUE TITLE
Add premium selection

### ISSUE THREAD

**Issue by ChaykeeMor** [ji-devs/ji-cloud#3540]
We need a way to mark our JIGs for premium access.

**Comment by corinnewo** [comment]
Just as we have a checkbox in the image backend for premium, we need to add in the checkbox to the JIG curation page (possibly even within a column of the table) to mark it as premium.
Then this field would control what would be visible to subscribers.
@MendyBerger Is there a field in the backend for this? There should be as it was one of the original requirements.

![image](https://user-images.githubusercontent.com/16290361/211805027-bb820373-24e0-4d7f-a8db-67d47acddc37.png)

**Comment by corinnewo** [comment]
We also need the curation for courses and resources!

**Comment by ChaykeeMor** [comment]
I don't think we need it for courses since individual jigs will be marked so if there's a course with some free and some premium jigs...you'll have access to only the jigs in the course that are available to your subscription.

**Comment by ChaykeeMor** [comment]
and resources we can blanket... either all free or all premium. Unless we add resources to our current curation table.

**Comment by MendyBerger** [comment]
Yes, that field exists
